.github/workflows/starlette.yml

```
@@ -0,0 +1,32 @@
+name: starlette_test_suite
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+    branches: ["master"]
+
+jobs:
+  starlette-testsuite-0_14_2:
+    runs-on: "ubuntu-latest"
+    defaults:
+      run:
+        working-directory: starlette
+    steps:
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+      - uses: actions/checkout@v2
+        with:
+          path: ddtrace
+      - uses: actions/checkout@v2
+        with:
+          repository: encode/starlette
+          ref: 0.14.2
+          path: starlette
+      - name: Install ddtrace
+        run: pip install ../ddtrace
+      - name: Install dependencies
+        run: "scripts/install"
+      #Parameters for keyword expression skip 3 failing tests that are expected due to asserting on headers. The errors are because our context propagation headers are being added
+      - name: Run tests
+        run: ddtrace-run pytest -p no:warnings --ddtrace-patch-all tests -k 'not test_request_headers and not test_subdomain_route and not test_websocket_headers'
```

```suggestion
        run: pytest -p no:warnings --ddtrace-patch-all tests -k 'not test_request_headers and not test_subdomain_route and not test_websocket_headers'
```

This is redundant with the `--ddtrace-patch-all` option